### DISCUSSION TITLE
Setting up email

### DISCUSSION THREAD

**Discussion by TryCatchCrash** [wger-project/docker#33] [Q&A]
I've set up prod.env for email (i know the settings are fine since i have a non-docker instance running and sending mails with the same settings) and it seems to ignore the FROM_EMAIL setting.

I see in the wger_server log:
smtplib.SMTPDataError: (553, b'Relaying disallowed as **wger@example.com**')

FROM_EMAIL is set to admin@<mydomain.ext>

Am I doing something wrong or is this a bug?

**Comment by rolandgeider** [comment]
mhhh, that should be enough, but I'll check just in case

**Reply by rolandgeider** [reply]
That should work, I'm creating an issue from this so it doesn't get lost

**Reply by TryCatchCrash** [reply]
Thanks for looking into it.
